### PR TITLE
[21.09] Fix containerized NCBI Datasets Genomes data source tool

### PR DESCRIPTION
**Pull request by mvdbeek**
It was missing ca-certificates and a small hack necessary for go programs to find the certificates.

## How to test the changes?
(Select all options that apply)
- [ ] I've included appropriate [automated tests](https://docs.galaxyproject.org/en/latest/dev/writing_tests.html).
- [ ] This is a refactoring of components with existing test coverage.
- [ ] Instructions for manual testing are as follows:
  1. [add testing steps and prerequisites here if you didn't write automated tests covering all your changes]

## License
- [x] I agree to license these contributions under [Galaxy's current license](https://github.com/galaxyproject/galaxy/blob/dev/LICENSE.txt).
- [x] I agree to allow the Galaxy committers to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT). If this condition is an issue, uncheck and just let us know why with an e-mail to galaxy-committers@lists.galaxyproject.org.

### CODE REVIEW

**Review by BradHolmes — APPROVED** [review]
Thanks Marius!  Looks great to me.

File: tools/data_source/ncbi_datasets.xml [hunk 1]
```diff
@@ -5,9 +5,12 @@
     </edam_operations>
     <requirements>
         <requirement type="package" version="12.11.0">ncbi-datasets-cli</requirement>
+        <requirement type="package" version="2021.10.8">ca-certificates</requirement>
```
Comment (bgruening) [inline tools/data_source/ncbi_datasets.xml]: shouldn't this be in the conda package?

File: tools/data_source/ncbi_datasets.xml [hunk 2]
```diff
@@ -5,9 +5,12 @@
...
     </requirements>
     <version_command>datasets version</version_command>
     <command><![CDATA[
+## If running in container use certificate from ca-certificates instead of outdated / missing container certificates
+[ -f /usr/local/ssl/cacert.pem ] && export export SSL_CERT_FILE="/usr/local/ssl/cacert.pem";
```
Comment (hexylena) [inline tools/data_source/ncbi_datasets.xml]:
```suggestion
[ -f /usr/local/ssl/cacert.pem ] && export SSL_CERT_FILE="/usr/local/ssl/cacert.pem";
```

File: tools/data_source/ncbi_datasets.xml (hunk 1 above)
Comment (mvdbeek) [reply tools/data_source/ncbi_datasets.xml]: Yeah, I agree, will open a PR.

Comment (mvdbeek) [reply tools/data_source/ncbi_datasets.xml]: Actually I already did https://github.com/conda-forge/ncbi-datasets-cli-feedstock/pull/1

**Review by jdavcs — APPROVED** [review]

File: tools/data_source/ncbi_datasets.xml (hunk 1 above)
Comment (mvdbeek) [reply tools/data_source/ncbi_datasets.xml]:
```suggestion
```

**Review by bgruening — APPROVED** [review]

**Pull request merged**